template/web/src/containers/Head.tsx

```diff
@@ -60,17 +60,17 @@ const Head = (props: HeadProps): JSX.Element => {
 
   return (
     <Helmet
-      titleTemplate={`%s — ${defaultMeta?.metaTitle}`}
-      defaultTitle={defaultMeta?.metaTitle || title}
-      {...props}
+    //titleTemplate={`%s — ${defaultMeta?.metaTitle}`}
+    //defaultTitle={defaultMeta?.metaTitle || title}
+    // {...props}
     >
       <html lang="en" />
 
       <style type="text/css">{}</style>
 
-      <meta name="robots" content={indexSafe} />
+      {/* <meta name="robots" content={indexSafe} />
 
-      <meta name="description" content={meta?.metaDescription || defaultMeta?.metaDescription} />
+  <meta name="description" content={meta?.metaDescription || defaultMeta?.metaDescription} /> */}
```

Comment:
why are all of the meta tags removed? do they not work? how did we handle SEO for ARYS?

Reply:
idk --> I just did the same as you did for arys before development. 
In Arys everything was added again